cve_bin_tool/cli.py

```diff
@@ -178,7 +201,12 @@ def main(argv=None):
                           """
         LOGGER.warning(warning_nolinux)
 
-    if not os.path.isfile(args.directory) and not os.path.isdir(args.directory):
+    if not args.directory and not args.input_file:
+        LOGGER.error("Error: Either a directory to scan or an input file required")
```

Comment:
```suggestion
        LOGGER.error("Please specify a directory to scan or an input file.")
```
We don't need to say "error" because the logging system does that.  (these messages were written before the logging system, which is why the others say it, but we can probably clean those up)